erpnext/accounts/report/cash_flow/custom_cash_flow.py

```diff
@@ -124,27 +124,27 @@ def setup_mappers(mappers):
 		)
 
 		for label in account_types_labels:
-			names = [d['name'] for d in account_types if d['label'] == label[0]]
+			names = [d['account_name'] for d in account_types if d['label'] == label[0]]
```

Comment:
`names` are used to query the `gl entries`, it was originally `child_row.name` which now is changed to `child_row.account`